tests/console/ping.pm

```diff
@@ -0,0 +1,34 @@
+# SUSE's openQA tests
+#
+# Copyright 2016-2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: very basic ping IPv6 test.
+# Maintainer: Petr Vorel <pvorel@suse.cz>
+
+use base "consoletest";
+use testapi;
+use strict;
+use warnings;
+
```

Comment:
It is better to use `Mojo::Base 'consoletest'` as it automatically includes `string and warnings`